bitops.schema.yaml

```diff
@@ -31,6 +31,10 @@ bitops:
           type: string
           description: "Path to log folder"
     
+    installed_plugins_dir:
+      type: sring
+      description: "Name of the directory where plugins will be installed"
```

Comment:
Do we really need to expose this new parameter in the bitops config?

Any examples where the user would need to change the `installed_plugins_dir` from the new default value `installed-plugins`? I understand it solves an issue as long the dir name is unique.

Trying to find the balance here on how exposing this variable will help.
Every new setting brings overload and configuration complexity to the user's shoulders.

Reply:
+1 for removing it. @mickmcgrath13 thoughts?

Reply:
I'm good removing this as config and just hardcoding it